services/navigation/builtin/builtin_test.go

```diff
@@ -574,6 +575,27 @@ func TestNavSetup(t *testing.T) {
 	paths, err := ns.Paths(ctx, nil)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, paths, test.ShouldBeEmpty)
+
+	test.That(t, len(ns.(*builtIn).boundingRegions), test.ShouldEqual, 1)
+	test.That(t, len(ns.(*builtIn).obstacles), test.ShouldEqual, 1)
+}
+
+func TestNavSetUpFromFaultyConfig(t *testing.T) {
```

Comment:
I don't know what is meant by faulty config.
From what I can see none of these tests encounter an error condition. What is faulty about that?

Reply:
> From what I can see none of these tests encounter an error condition

Not entirely sure what you mean since we do
```
		_, err = navigation.FromRobot(myRobot, "test_navigation")
		test.That(t, err, test.ShouldNotBeNil)
```

I can update this to say
```
		_, err = navigation.FromRobot(myRobot, "test_navigation")
		test.That(t, err, test.ShouldNotBeNil)
                 test.That(t, strings.Contains(err.Error(), "unsupported Geometry type"), test.ShouldBeTrue)
```
so that it is more explicit what was wrong.
Is this what you mean?

Reply:
yea, lets test for the actual error we expect to get